src/bitcoin-chainstate.cpp

```diff
@@ -80,6 +80,10 @@ int main(int argc, char* argv[])
     const ChainstateManager::Options chainman_opts{
         .chainparams = chainparams,
         .adjusted_time_callback = static_cast<int64_t (*)()>(GetTime),
+        .block_tree_db_opts = {
+            .db_path = gArgs.GetDataDirNet() / "blocks" / "index",
```

Comment:
In commit "CBlockTreeDB: Add ::Options, init with BlockManager" (961c86a113875366154175677ca4acc326c49e79)

Should probably use `abs_datadir` variable, not `gArgs`